spec/schema.json

```diff
@@ -145,7 +145,8 @@
         "INVALID_COMPANY_TYPE_CS01_FILING_NOT_REQUIRED",
         "INVALID_COMPANY_TYPE_PAPER_FILING_ONLY",
         "INVALID_COMPANY_TYPE_USE_WEB_FILING",
-        "INVALID_OFFICER_COUNT"
+        "INVALID_OFFICER_COUNT",
+        "MULTIPLE_PSCS_FOUND"
```

Comment:
Last comment I promise, maybe we should have this response as "INVALID_COMPANY_OFFICERS_MORE_THAN_ONE_PSC"
So somebody seeing just that error response would be able to identify exactly why it has failed.

Just "MULTIPLE_PSCS" a user wouldn't be able to identify what the correct number of PSC's is.

Reply:
INVALID_COMPANY_APPOINTMENTS_MORE_THAN_ONE_PSC perhaps?

Reply:
Yes that looks better, go for that one